docs/boshdirector.adoc

```diff
@@ -28,114 +40,307 @@ Click on the zone; you'll notice an NS-type entry for the subdomain, as shown be
 .Cloud DNS view of your hosted zone
 image::hosted_zone.png[Cloud DNS]
 
--> Communicate the value of the _Data_ column to your instructor (which should be a list of google name servers).
+-> Communicate the value of the _Data_ column to your instructor (which should be a list of Google name servers).  Your instructor will add these name servers to the top-level domain DNS record.
 
 Shortly after the NS record is created, you should be able to verify proper DNS resolution for any host under your subdomain using a client such as `dig` or `nslookup`.
 
 
 == Instructions
 
+You can refer to the  http://docs.pivotal.io/pivotalcf/customizing/gcp-om-config.html[Configuring Ops Manager^] instructions from the PCF documentation as you complete this lab.  As a general guideline, the lab instructions will specify the parameters to override - otherwise you should accept default values.
+
+=== Access Ops Manager
+
 The Ops Manager runs a web server that is accessible at https://pcf.{env_name}.{domain_name}/[^].
 
 . Open a web browser and visit that page.
++
+[.thumb]
+.Ops Manager Home Page
+image::authentication-system.png[Ops Manager Home Page]
 
-. follow the http://docs.pivotal.io/pivotalcf/customizing/gcp-om-config.html[Configuring Ops Manager^] instructions from the PCF documentation
+. Select the *Internal Authentication* method when setting up access to your Ops Manager.
++
+[.thumb]
+.Internal Authentication
+image::authentication-details.png[Internal Authentication]
```

Comment:
Don't the official instructions already provide these screenshots?  are we duplicating the docs here?